CHANGELOG.md

```diff
@@ -52,6 +52,7 @@
 * [ENHANCEMENT] Store-gateway: record index header loading time separately in `cortex_bucket_store_series_request_stage_duration_seconds{stage="load_index_header"}`. Now index header loading will be visible in the "Mimir / Queries" dashboard in the "Series request p99/average latency" panels. #5011 #5062
 * [ENHANCEMENT] Querier and ingester: add experimental support for streaming chunks from ingesters to queriers while evaluating queries. This can be enabled with `-querier.prefer-streaming-chunks=true`. #4886
 * [ENHANCEMENT] Update Docker base images from `alpine:3.17.3` to `alpine:3.18.0`. #5065
+* [ENHANCEMENT] Compactor: reduced the number of object storage `HEAD` API calls issued when syncing block's `meta.json` files. #5063
```

Comment:
It may be HEAD for one storage, GET in another. I would be less specific... We can say "reduction of exists" operations, as that's what is visible in our bucket metrics.

Reply:
Yes, done.